frameworks/cassandra/tests/test_racks.py

```diff
@@ -43,7 +43,7 @@ def test_rack():
     log.info("node: {}".format(node))
 
     assert node.get_rack() != "rack1"
-    assert "us-west" in node.get_rack()
+    assert any(rack in node.get_rack() for rack in config.AWS_REGIONS)
```

Comment:
This checks if the node is on a rack in _any_ region, not specifically if the node is on a rack in the _same_ region as the cluster. I was thinking more along the lines of determining what region the cluster is in eg `us-east-1` and then verifying that the rack is indeed in `us-east` as well.

Look through the python `sdk_` utils in `testing/` or `tools/` directories. There might be something that can help you with that. Or maybe create your own helper method to determine cluster region!

Reply:
I have written `get_cluster_region` helper method in `sdk_utils` to get the region of the cluster.